### PR TITLE
Issue/269/create success message in set map position view

### LINKED ISSUE #269: Create success message in set map position view
+ Fix error with button position

### PR DESCRIPTION
**Pull request by Bedpaw**
closes #269

### CODE REVIEW

File: src/components/molecules/banner-map.vue [hunk 1]
```diff
@@ -19,5 +33,28 @@ import AButtonPrimary from 'atoms/button/primary';
 export default {
   name: 'm-banner-map',
   components: { AButtonPrimary },
+  data: () => ({
+    showSuccessMessage: false,
+  }),
+  props: {
+    successMessageDuration: {
+      type: Number,
+      default: 1500,
```
Comment (dbetka) [inline src/components/molecules/banner-map.vue:42]: I think that 2000 ms will be better.

File: src/components/molecules/banner-map.vue [hunk 2]
```diff
@@ -1,15 +1,29 @@
...
-    </a-button-primary>
+  <div>
+    <transition name="fade">
+      <div v-if="showSuccessMessage === false" class="m-banner-map">
+        <div class="f-flex-1">
+          <slot name="text"></slot>
+        </div>
+        <a-button-primary
+          class="f-flex-0 f-m-0"
+          :add-class="'min'"
```
Comment (dbetka) [inline src/components/molecules/banner-map.vue:10]: Remove `:` and `'` chars.

File: src/components/molecules/banner-map.vue [hunk 3]
```diff
@@ -1,15 +1,29 @@
...
+          <slot name="button-name"></slot>
+        </a-button-primary>
+      </div>
+    </transition>
+    <transition name="fade">
+      <div v-if="showSuccessMessage"
+           class="m-banner-map f-message-success">
+        <span class="f-text-center">
+          <slot name="success-message">
+            Nowa pozycja została zapisana.
```
Comment (dbetka) [inline src/components/molecules/banner-map.vue:22]: Add to dictionary.

File: src/components/molecules/banner-map.vue [hunk 4]
```diff
@@ -19,5 +33,28 @@ import AButtonPrimary from 'atoms/button/primary';
...
+  props: {
+    successMessageDuration: {
+      type: Number,
+      default: 1500,
+    },
+  },
+  methods: {
+    emitSuccessMessage () {
+      this.showSuccessMessage = true;
+      window.setTimeout(() => {
```
Comment (dbetka) [inline src/components/molecules/banner-map.vue:48]: `window` is unnecessary.

File: src/components/organisms/admin/set-map-position.vue [hunk 5]
```diff
@@ -20,7 +20,10 @@ export default {
   methods: {
     onSavePosition () {
       this.$refs.oMap.saveLastMapPosition();
-      this.$store.dispatch('event/updateEvent');
+      this.$store.dispatch('event/updateEvent')
+        .then(() => {
```
Comment (dbetka) [inline src/components/organisms/admin/set-map-position.vue:24]: It could be shorter.
```
.then(() => this.$refs.banner.emitSuccessMessage());
```

**Review by dbetka — CHANGES_REQUESTED** [review]
To change

File: src/components/molecules/banner-map.vue (hunk 1 above)
Comment (Bedpaw) [reply src/components/molecules/banner-map.vue:42]: ok

File: src/components/molecules/banner-map.vue (hunk 2 above)
Comment (Bedpaw) [reply src/components/molecules/banner-map.vue:10]: done

File: src/components/molecules/banner-map.vue (hunk 3 above)
Comment (Bedpaw) [reply src/components/molecules/banner-map.vue:22]: done

File: src/components/molecules/banner-map.vue (hunk 4 above)
Comment (Bedpaw) [reply src/components/molecules/banner-map.vue:48]: +

File: src/components/organisms/admin/set-map-position.vue (hunk 5 above)
Comment (Bedpaw) [reply src/components/organisms/admin/set-map-position.vue:24]: done

**Review by dbetka — APPROVED** [review]

**Pull request merged**